microsoft-365/admin/manage/room-and-equipment-mailboxes.md

```diff
@@ -101,8 +101,8 @@ Open the meeting you scheduled in Outlook and then remove the room or equipment
   
 ### Does a room mailbox or equipment mailbox need a product license?
 
-No. While most people in your organization need a license to use Microsoft 365, you don't need to assign a license to a room mailbox or equipment mailbox.
+No. While most people in your organization need a license to use Microsoft 365, you don't need to assign a license to a room mailbox or equipment mailbox, in case the resource mailbox needs to have a size bigger than 50 GB, or, in case some features are needed (Retention policies from Security and Compliance, or, Hold from Exchange Online), or, in case the resource mailbox doesn't need an Online Archive attached to it.
```

Comment:
```suggestion
No. While most people in your organization need a license to use Microsoft 365, you don't need to assign a license to a room mailbox or equipment mailbox. However, should more than 50 GB of storage space be required for a mailbox to accommodate additional features such as retention policies from security & compliance, Litigation Hold from Exchange Online, you need a license assigned to the mailbox.
```

Reply:
@dimcry - I am really not sure about this edit. If you don't agree with it, please rephrase your original suggestion, because as it stands it is difficult to understand. Many thanks.

Reply:
@JohanFreelancer9 the explanation should be similar to the one we have for Shared MBXes (in the Exchange Online Limits article). Also, the Exchange Online limits article (and any other that contain similar information) should be updated, also, with this information in regards to Resource MBXes. Resource MBXes are similar to Shared MBXes from the license perspective...



From the Exchange Online limits (https://docs.microsoft.com/en-us/office365/servicedescriptions/exchange-online-service-description/exchange-online-limits?source=docs):
---
To access a shared mailbox, a user must have an Exchange Online license, but the shared mailbox doesn't require a separate license. Without a license, shared mailboxes are limited to 50 GB. To increase the size limit to 100 GB, the shared mailbox must be assigned an Exchange Online Plan 2 license or an Exchange Online Plan 1 license with an Exchange Online Archiving add-on license. This will also let you enable auto-expanding archiving for an unlimited amount of archive storage capacity. Similarly, if you want to place a shared mailbox on litigation hold, the shared mailbox must have an Exchange Online Plan 2 license or an Exchange Online Plan 1 license with an Exchange Online Archiving add-on license. If you want to apply advanced features such as Microsoft Defender for Office 365, Advanced eDiscovery, or automatic retention policies, the shared mailbox must be licensed for those feature(s).
---
---
Resource mailboxes don't require a license. However, without a license, resource mailboxes are limited to 50 GB. To increase the mailbox size, an E3 or E5 license must be assigned. This will increase the mailbox to 100 GB.
---
---
Archive mailboxes can only be used to archive mail for a single user or entity (such as a shared mailbox) for which a license has been applied. Using archive mailboxes as a means of storing mail from multiple users or entities is prohibited. For example, an IT administrator can't create a shared mailbox and have users copy it (through the Cc or Bcc field, or through a transport rule) for the explicit purpose of archiving. Note that a shared mailbox that multiple people use does not actually store email for those individual users. Multiple users have access, and they send email as the shared mailbox. Therefore, the only emails stored in the shared mailbox are those sent to or from it, as the shared mailbox.
---
---
Shared and resource mailboxes don't require a license. However, without a license, these mailboxes are limited to 50 GB. To increase the mailbox size, an E3 or E5 license must be assigned. This will increase the mailbox to 100 GB.
---